website/docs/concepts/html/lists.md

```diff
@@ -70,9 +40,82 @@ html! {
     </ul>
 };
 ```
+
 <!--END_DOCUSAURUS_CODE_TABS-->
 
+## Keyed lists
+
+A keyed list is a very optimized list that has keys on all tags.
+A `key` gives an html element a unique identifier which Yew can use for optimization purposes.
+
+:::warn
+
+Key has to to be unique and has to not dependent on the order of the list.
+
+:::
+
+Generally it is recommended to always add keys to the lists.
+
+Most of the time, to add keys, you will need to write something like this:
+
+```rust , ignore
+use yew::{html};
+
+let names = vec!["Sam","Bob","Ray"]
+
+html! {
+    <div id="introductions">
+        {
+            names.into_iter().map(|name| {
+                html!{<div key={name}>{format!("Hello, I'am {}!",name)}</div>}
```

Comment:
```suggestion
                html!{<div key={name}>{ format!("Hello, I'am {}!",name) }</div>}
```
I think the normal style for our docs and examples is to have a space padding inside the curly braces